platform/darwin/src/MGLStyle.h

```diff
@@ -200,7 +200,11 @@ static const NSInteger MGLStyleDefaultVersion = 9;
 /**
  Adds a new source to the current style.
 
- @note Adding the same source instance more than once will result in a 
+ @note Wait until the map style has finished loading before adding a new source.
+    Implement `MGLMapViewDelegate` methods `-mapViewDidFinishLoadingMap:`
+    or `-mapView:didFinishLoadingStyle:`
```

Comment:
Nit: end the sentence with a period. Also, two `@note`s in a row is a bit much; I don’t think this paragraph needs that treatment.